.github/workflows/release.yml

```diff
@@ -14,19 +14,30 @@ permissions:
   packages: write
 
 jobs:
-  trimTagRef:
+  tagList:
     runs-on: ubuntu-latest
     outputs:
-      TRIMMED_REF_NAME: ${{ steps.trim.outputs.TRIMMED_REF_NAME }}
+      tags: ${{ steps.tags.outputs.OUTPUT_TAGS }}
     steps:
-      - name: Trim tag ref
-        id: trim
-        run: echo "TRIMMED_REF_NAME=${REF_NAME#v}" >> "$GITHUB_OUTPUT" && cat "$GITHUB_OUTPUT"
+      - name: Get image tag list
+        id: tags
+        run: |
+          set -eux -o pipefail
```

Comment:
do we want to make this a default (except debug of course) by using `shell: bash` everywhere?

Reply:
That's fine, though I'd probably still make it explicit.

Reply:
Added `shell: bash`